infra/tf/k8s_infra/nomad.tf

```diff
@@ -59,6 +59,8 @@ locals {
 		EOT
 	}
 	nomad_checksum_configmap = sha256(jsonencode(local.nomad_server_configmap_data))
+
+	# Recommendations: https://developer.hashicorp.com/nomad/docs/install/production/requirements#resources-ram-cpu-etc
 	service_nomad = lookup(var.services, "nomad", {
 		count = 1
 		resources = {
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 63 to 71]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This document seems to recommend might higher resources than is shown here for Nomad servers, unless I'm not looking in the right place for server resource allocation.

> We suggest having between 4-8+ cores, 16-32 GB+ of memory<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/369?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->